etcdctl/ctlv3/ctl_cov.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2017 The etcd Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// +build cov
+
+package ctlv3
+
+import (
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/coreos/etcd/etcdctl/ctlv3/command"
+)
+
+func Start() {
+	rootCmd.SetUsageFunc(usageFunc)
+
+	// Make help just show the usage
+	rootCmd.SetHelpTemplate(`{{.UsageString}}`)
+	argsStr := os.Getenv("ETCD_CTL_ARGS")
+	if argsStr == "" {
+		command.ExitWithError(command.ExitError, fmt.Errorf("ETCD_CTL_ARGS not set"))
+	}
+	args := strings.Split(argsStr, " ")
+	rootCmd.SetArgs(args[1:])
```

Comment:
`args[1:]` here is important because `ETCDCTLCOV_ARGS=etcdctl_test arg1 arg2 arg3 ...`. the `SetArgs` only wants a`arg1 arg2 arg3 ...`. Should I inline it like `rootCmd.SetArgs(strings.Split(os.Getenv("ETCDCTLCOV_ARGS", " ")[1:])`?